lib/service/mix_panel_client_service.dart

```diff
@@ -107,4 +105,16 @@ class MixPanelClientService {
       log(e.toString());
     }
   }
+
+  void setLabel(String prop, dynamic value) {
+    mixpanel.getPeople().set(prop, value);
+  }
+
+  dynamic getConfig(String name) {
```

Comment:
can you specify the type of function & value here, `dynamic` look hard for other people to know what it is.